BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/Collection/OrchestraCollection.js

```diff
@@ -23,10 +23,12 @@ class OrchestraCollection extends Backbone.Collection
      * 'update': '/user/update',
      * 'delete': '/user/remove'
      * }
+     * @param {object} options - sync options
+     *
      * @returns {Object}
      * @private
      */
-    _getSyncUrl() {
+    _getSyncUrl(options) {
```

Comment:
@alavieille : is it possible to have explicits paramters rather than a bag here ?